.github/workflows/test.yml

```diff
@@ -158,7 +156,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          apt-get -y install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb netcat lsof
+          apt-get -qy install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb netcat-openbsd lsof
```

Comment:
These should probably just be added to the image. It only adds 600kb for these on the existing (old) canon image so I assume it'll probably be similar.

Reply:
Still think this should probably be built into the image, since the e2e tests run pretty much every PR update anyway, but it can wait for a later PR (set)

Reply:
made a follow-up ticket here https://viam.atlassian.net/browse/APP-2906